packages/marko-web-theme-monorail/components/marko.json

```
@@ -21,6 +21,9 @@
   "<theme-client-side-block-loader>": {
     "template": "./client-side-block-loader.marko"
   },
+  "<theme-facebook-connect>": {
+    "template": "./facebook-connect.marko"
```

This should accept an ids prop:
```suggestion
    "template": "./facebook-connect.marko",
    "@ids": "array"
```